xds/internal/client/v2/client_test.go

```diff
@@ -236,7 +236,9 @@ var (
 		},
 		TypeUrl: version.V2RouteConfigURL,
 	}
-	emptyRouteConfig             = &xdspb.RouteConfiguration{}
+	emptyRouteConfig = &xdspb.RouteConfiguration{
```

Comment:
this is not empty any more :)

Reply:
Renamed to `noVirtualHostsRouteConfig`, because it's only used by `noVirtualHostsInRDSResponse`